numpy/core/tests/test_multiarray.py

```diff
@@ -7161,6 +7161,7 @@ def test_max_dims(self):
         a = np.empty((1,) * 32)
         self._check_roundtrip(a)
 
+    @pytest.mark.slow
     def test_error_too_many_dims(self):
```

Comment:
I assume this test is slow because of the repeated calls to `gc.collect()`?

Reply:
I did not dive into the profiling, but it makes sense.